lib/arclight/custom_document.rb

```diff
@@ -44,13 +44,15 @@ def to_solr(solr_doc = {})
       super
       solr_doc['id'] = eadid.first.strip.tr('.', '-')
       arclight_field_definitions.each do |field|
-        Solrizer.insert_field(solr_doc, field[:name], field[:value], field[:index_as])
+        # we want to use `set_field` rather than `insert_field` since we may be overriding fields
+        Solrizer.set_field(solr_doc, field[:name], field[:value], field[:index_as])
```

Comment:
I think this is 👍.  I believe in every case we've been assuming we are overriding the field entirely.